samza-core/src/main/java/org/apache/samza/operators/impl/OperatorImpl.java

```diff
@@ -82,16 +81,15 @@
   private EndOfStreamStates eosStates;
   // watermark states
   private WatermarkStates watermarkStates;
-  private TaskContext taskContext;
+  private CallbackScheduler callbackScheduler;
   private ControlMessageSender controlMessageSender;
 
   /**
    * Initialize this {@link OperatorImpl} and its user-defined functions.
    *
-   * @param config  the {@link Config} for the task
-   * @param context  the {@link TaskContext} for the task
+   * @param context the {@link Context} for the task
```

Comment:
Update doc: "the {@link Context} for the application", here and everywhere else.